src/keyshot_command_handler.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python3
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""
+This script runs inside KeyShot. It accepts JSON commands from STDIN and executes them.
+"""
+
+import json
+import sys
+import traceback
+import lux  # noqa: F401
```

Comment:
Quick Question - If we wanted to add new KeyShot-specific functionality in the future (like camera controls, material adjustments, etc), we'd primarily extend this file with new command handlers that call the appropriate `lux` API methods? :eyes:

Reply:
Yup, exactly!